.gitignore

```diff
@@ -0,0 +1,2 @@
+coverage
```

Comment:
My attempt to ignore the coverage files, as you can see it doesn't work so can get deleted